extensions/qcom/GLSL_QCOM_image_processing.txt

```diff
@@ -0,0 +1,231 @@
+	that these built-ins rely on separate samplers and textures as defined in GL_KHR_vulkan_glsl.
+
+    +------------------------------------+--------------------------------------------+
+    | Syntax                             | Description                                |
+    +------------------------------------+--------------------------------------------+
+    |   vec4 textureWeightedQCOM(        | weighted sample operation multiplies       |
+    |       sampler2D tex,               | a 2D kernel of filter weights with a corr- |
+    |       vec2      P,                 | esponding region of sampled texels and     |
+    |       sampler2DArray weight)       | sums the results to produce the output     |
+    |                                    | value.                                     |
+    +------------------------------------+--------------------------------------------+
+    |   vec4 textureBoxFilterQCOM(       | Linear operation taking average of pixels  |
+    |       sampler2D tex,               | within the spatial region described by     |
+    |       vec2      P,                 | boxSize.  The box is centered at coordinate|
+    |       vec2      boxSize)           | P and has width and height of boxSize.x    |
+    |                                    | and boxSize.y.                             |
+    +------------------------------------+--------------------------------------------+
+    |   vec4 textureBlockMatchSADQCOM(   | Block matching operation measures the      |
+    |       sampler2D target             | correlation (or similarity) of the target  |
+    |       uvec2     targetCoord,       | block and reference block.  targetCoord    |
+    |       sampler2D reference,         | and refCoord specify the bottom-left corner|
+    |       uvec2     refCoord,          | of the block in target and reference       |
+    |       uvec2     blockSize)         | images. The error metric is the Sum of     |
+    |                                    | Absolute Differences (SAD).                |
+    +------------------------------------+--------------------------------------------+
+    |   vec4 textureBlockMatchSSDQCOM(   | Block matching operation measures the      |
+    |       sampler2D target             | correlation (or similarity) of the target  |
+    |       uvec2     targetCoord,       | block and reference block.  TargetCoord    |
+    |       sampler2D reference,         | and refCoord specify the bottom-left corner|
+    |       uvec2     refCoord,          | of the block in target and reference       |
+    |       uvec2     blockSize)         | images. The error metric is the Sum of     |
+    |                                    | Square Differences (SSD).                  |
+    +------------------------------------+--------------------------------------------+
+
+    * <target> and <reference> specify the two input textures. If
```

Comment:
Here, for example, the text is different between ES and desktop, but it's not clear whether it's an intentional difference or just an editing problem.